src/main/webapp/gameDialogue.js

```diff
@@ -92,3 +93,14 @@ function modifyIfFinalStage() {
         }
       });
 }
+
+//play button functions
+function playbutton() {
+  if(document.getElementById('player-button').innerText == "PAUSE"){
```

Comment:
When you click this button title of the game shifts right/left to accomodate the new size of the added test. Let's fix that so it doesn't mess with positioning of other elements.

Reply:
I will work on the positioning of elements when the icons are added because to do it now would be counter productive.

Reply:
cool--add a TODO here to track it